.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
As required for this project, kindly create and ignore a `test.md` file. This will help you practice ignoring files that you wouldn't want to commit to GitHub which you'll find handy in the near future.